organization/_posts/2021/2021-03-04-coordination.md

```diff
@@ -0,0 +1,160 @@
+
+### Data Analysis Working Group Convenors
+
+Chris Burr (LHCb, CERN) has unfortunately had to step down as a convenor of DAWG. We are however delighted to announce that Nicole Skidmore (LHCb, U Manchester) has agreed to take this role.
+
+### C++ Course
+
+Next iteration, tentatively 30 August - 3 Sept. Pivot the material a bit to concentrate more on the on basics and to have more exercises.
+
+## Working Group Updates
+
+### Data Analysis
+
+- Working on metadata writeup to close out that discussion. Let us know if you want to help!
+- Now that Nicole has joined, we will start planning our next events.
+
+### Detector Simulation
+
+- Next meeting: Monday, March 8, on NEST (noble liquid simulation for dark matter, neutrino experiments)
+  - May 10: Simulating solid-state microphysics with Geant4 and G4CMP (Mike Kelsey)
+  - Considering subsequent meetings on other related topics
+- GPU usage:
+  - advertise next accelerator forum (March 10)
+  - possible meeting (April or end March) on Celeritas, Opticks integration w/ Geant4
+- ML for simulation: considering a few topical meetings
+  - Run 3 deployment plans and experiences (ATLAS, LHCb)
+  - Mathematical improvements in NN architectures: reweighting GANs, normalizing flows, ...
+  - Usage beyond LHC experiments (if any)
+
+### Reconstruction and Software Triggers
+
+- Feedback form to plan future events sent out (<https://forms.gle/sTM1cyERyKVZPFbE6>), to be filled out by March 12th
+- Kick-off meeting will take place on March 17th at 5 pm
+
+### PyHEP
+
+- 2nd topical meeting, on automatic differentiation with JAX, took place yesterday (<https://indico.cern.ch/event/985423/>).
+  - Good attendance with almost 40 participants.
+
+### Software Tools and Packaging
+
+- Had a nice meeting last week on European Environment for Scientific Software Installations (EESSI):
+  - Speaker: Bob Dröge (University of Groningen) 
+  - Agenda: <https://indico.cern.ch/event/1005954/>
+  - Graeme: Although they use Easibuild and we use LCGCMake/Spack, there are still many pieces in common (e.g., we could look at using `lmod` to setup releases)
+- Material from [GA's talk](https://indico.cern.ch/event/999543/) on performance monitoring should be made more widely available, a great resource:
+  - <https://indico.cern.ch/event/974382/contributions/4102981/attachments/2174921/3672328/linux-systems-performance.pdf>
+
+### Software Training
+
+- Quick presentation summarising recent GitHub CI/CD training <https://docs.google.com/presentation/d/12sS7p5E1skOAATztGk5C3NjK0FNCFV9VbWTVxX1UtME/edit?usp=sharing>
+
+### Event Generators
+
+- HL-LHC Review 2: Have been discussing the proposal from Liz and Graeme and have decided it would be good to dedicate some upcoming WG meetings to discuss the important topics - expecting to send announcement to the WG tomorrow.
```

Comment:
```suggestion
- HL-LHC Review 2: Have been discussing the proposal from Liz and Graeme and have decided it would be good to dedicate some upcoming WG meetings to discuss the important topics - expecting to send an announcement to the WG tomorrow.
```